src/index.html

```diff
@@ -61,7 +62,7 @@
           </ul>
         </div>
         <a href="http://sis-staff.berkeley.edu/timeline-links">
-          <img alt="Highlevel Timeline" class="timeline-banner" src="http://sis-staff.berkeley.edu/sites/default/files/intranet-timeline-button.jpg">
+          <img alt="Highlevel Timeline" class="timeline-banner" src="https://sis-staff.berkeley.edu/sites/default/files/intranet-timeline-button.jpg">
```

Comment:
@winstonkim will be interesting to see whether this actually works, seems like `https://sis-staff.berkeley.edu/` has an invalid certificate - created https://jira.berkeley.edu/browse/SISRP-7862 for Sam to follow up.